setup.py

```diff
@@ -194,6 +198,12 @@ def write_version_py(filename='qutip/version.py'):
         '_steadystate',
         '_stochastic',
     ],
+    'solver': [
+        '_feedback',
+        '_solverqevo',
+    ],
+    'solver.ode': [file.split("/")[-1][:-4]
+                   for file in glob("qutip/solver/*/*.pyx")],
```

Comment:
I suspect these won't work on Windows because the path separator there is not `/`. If that's important, we should fix it.

Reply:
I think it probably will work, but I think it would be better to use `os.path.split` and `os.path.join`